Content.Server/Speech/EntitySystems/MeleeSpeechSystem.cs

```diff
@@ -1,49 +1,84 @@
+using Content.Server.Actions;
 using Content.Server.Administration.Logs;
+using Content.Shared.Actions;
 using Content.Shared.Speech.Components;
 using Content.Shared.Speech.EntitySystems;
 using Content.Shared.Database;
-
+using Robust.Server.GameObjects;
+using Robust.Shared.Players;
 
 namespace Content.Server.Speech.EntitySystems;
 
 public sealed class MeleeSpeechSystem : SharedMeleeSpeechSystem
 {
 
-	[Dependency] private readonly IAdminLogManager _adminLogger = default!;
-
-
-	public override void Initialize()
+    [Dependency] private readonly IAdminLogManager _adminLogger = default!;
+    [Dependency] private readonly SharedActionsSystem _actionSystem = default!;
+    [Dependency] private readonly UserInterfaceSystem _uiSystem = default!;
+    public override void Initialize()
 	{
 		base.Initialize();
 		SubscribeLocalEvent<MeleeSpeechComponent, MeleeSpeechBattlecryChangedMessage>(OnBattlecryChanged);
-	}
-
+        SubscribeLocalEvent<MeleeSpeechComponent, MeleeSpeechConfigureActionEvent>(OnConfigureAction);
```

Comment:
Please don't mix tabs and spaces, use spaces.

Also leave newlines between fields and methods.